collect_app/src/main/java/org/odk/collect/android/projects/ProjectImporter.kt

```diff
@@ -0,0 +1,19 @@
+package org.odk.collect.android.projects
+
+import org.odk.collect.android.preferences.keys.MetaKeys
+import org.odk.collect.android.preferences.source.Settings
+
+class ProjectImporter(
+    private val projectsRepository: ProjectsRepository,
+    private val metaSettings: Settings
+) {
+    fun importDemoProject() {
+        val project = Project("Demo project", "D", "#3e9fcc", "1")
```

Comment:
I implemented it that way to identify the default project somehow. Thanks to that we can assume that if on the list of projects there is one with uuid 1 it's the default project.
This might help us for example to manage preferences. For all the other projects to access their preferences you would use their uuid but you can't do the same for the default project since it should use default preferences.

Reply:
Hmmm that's a good point. Maybe we should use a more human-readable string like "DEMO" then? Making it a constant is probably a good move as well.

Reply:
I'm not sure if I understand... "DEMO" just for project name? or what?

Reply:
Sorry I meant for the ID.

Reply:
Ah yeah that's a good idea. Added.